docs/sphinx/user_guide/feature/loop_basic.rst

```diff
@@ -325,7 +325,7 @@ threads within the same team. The *RAJA Teams* abstraction consist of three main
     loop execution or OpenMP threaded regions.
 
 The team loop interface combines concepts from ``RAJA::forall`` and ``RAJA::kernel``.
-Various policies from ``RAJA::kernel`` are compatible with the ``RAJA Teams``
+Various policies from ``RAJA::kernel`` are compatible with the ``RAJA::launch``
```

Comment:
I noticed that we mix RAJA::expt::launch and RAJA::launch. But I think @ajkunen made a change so that we can use experimental features without using the expt:: name space?

Reply:
For the release, let's move launch stuff out of the 'expt' namespace and fix up the docs.